db/constants.py

```diff
@@ -1,3 +1,4 @@
 MATHESAR_PREFIX = "mathesar_"
 ID = "id"
 ID_ORIGINAL = "id_original"
+INFERENCE_SCHEMA = f"{MATHESAR_PREFIX}temp_schema"
```

Comment:
Should we name this `mathesar_inference_schema` for clarity?

Reply:
Good idea.  Mindless cut-and-paste on my part, sorry.